content/modules/redisgears/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: RedisGears
+description:
+weight: 55
+alwaysopen: false
+categories: ["Modules"]
+---
+## What is RedisGears?
+
+RedisGears is an engine for data processing in Redis. RedisGears supports batch and event-driven processing for Redis data. To use RedisGears, you write functions that describe how your data should be processed. You then submit this code to your Redis deployment for remote execution.
+
+As of v1.0.0, code for RedisGears must be written in Python. However, an internal C API exists and can be used by other Redis modules. In addition, support for other languages, including Java and JavaScript, is being planned.
```

Comment:
Let's not add this `In addition, support for other languages, including Java and JavaScript, is being planned.` but let's generalise it to the fact that we want to add new languages soon